### DISCUSSION TITLE
Pester V5 support

### DISCUSSION THREAD

**Discussion by SQLDBAWithABeard** [sqlcollaborative/dbachecks#881] [Ideas]
Take a look at the pesterv5 branch

**Comment by SQLDBAWithABeard** [comment]
Improving AllInstanceInfo performance
https://user-images.githubusercontent.com/6729780/165101666-7c7a3451-581d-49be-9e3e-12e0926f887d.mp4

**Comment by SQLDBAWithABeard** [comment]
Invoke-DbcCheck running in Pester v5
https://user-images.githubusercontent.com/6729780/165101742-dcc695a7-48cd-4cad-a720-7664a0bbbdf9.mp4

**Comment by SQLDBAWithABeard** [comment]
Invoke-DbcCheck Performance improvement running in v5 and with AllInstanceInfo Improvements 
https://user-images.githubusercontent.com/6729780/165101803-d27cf7b0-9ce7-460d-a03f-7b89b692015e.mp4

**Comment by wsmelton** [comment]
Is there any timeline on when the module will be released with Pester v5 support?
